h/static/styles/share-link.scss

```diff
@@ -27,6 +33,30 @@
   width: 100%;
 }
 
+.share-target {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  border-bottom: 1px solid $gray-lighter;
+  font-size: $body1-font-size;
+  padding: 10px;
+}
+
+.share-target__label {
+  font-weight: bold;
+}
+
+.share-target__icon {
+  color: #7A7A7A;
+  text-decoration: none;
+  font-size: 20px;
+  cursor: pointer;
+}
+
+.share-target__icon:hover {
+  color: #202020 !important;
```

Comment:
What is `!important` for here? - That is something we should try to avoid as much as possible in the app.